lib/typescript/render/components/ImageWithFallback/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useEffect, useState} from 'react';
+
+type ImageRenderProps = {
+  src: string;
+  alt?: string;
+  className?: string;
+};
+
+/**
+ * This is a global list of images that failed to load.
+ * Sadly the render component is not able to fix wrong payloads in the
+ * redux store and this is the only way for it to remember failed states
+ * and not start flickering on every redraw of the messages
+ */
+const failedUrls = [];
```

Comment:
This reduces a lot of the weird flickering behaviour because of redraws. If we fix the redraws, this can be gone. See #1415 for details.